roles/kubernetes-apps/ansible/templates/dns-autoscaler.yml.j2

```diff
@@ -16,20 +16,20 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: dns-autoscaler
+  name: dns-autoscaler{%- if dns_mode in ['coredns', 'coredns_dual'] -%}{{ coredns_ordinal_suffix | default('') }}{%- endif -%}
```

Comment:
just: 
```
dns-autoscaler{{coredns_ordinal_suffix|default('')}}
```
should be enough

Reply:
Also if `coredns_ordinal_suffix` is unset?

Reply:
it will default to ''.